tutorial/01_Conditions_Loops_Data_Types.ipynb

```diff
@@ -721,6 +721,78 @@
     "# TODO: add an example from the weather app with the same variables\n",
     "# TODO: create a dictionary with your details and print them"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97fe17b1-3cd5-48a1-a8fd-6ac974f3a291",
+   "metadata": {},
+   "source": [
+    "# Bonus!\n",
+    "\n",
+    "Hey! you made it this far! first of all, good job!\n",
+    "Are you bored? feeling adventurous? let's try another bonus :)\n",
+    "\n",
+    "Try to write Python code that prints a triangle with stars, such that its size is based on a given input n.\n",
+    "for example, for n=3\n",
+    "*\n",
+    "* *\n",
+    "\n",
+    "for n = 1\n",
+    "*\n",
+    "\n",
+    "for n = 5\n",
+    "*\n",
+    "* *\n",
+    "* * *\n",
+    "* * * *\n",
+    "* * * * *\n",
+    "\n",
+    "etc..\n",
+    "\n",
+    "hint: as developers, when encountring a big problem we usually solve it by first dividing it into smaller problems.\n",
+    "so maybe try to think of how you would print each line first, and then move to printing all the lines,\n",
+    "for example, for the 3rd line, you need to print 3 stars right?\n",
+    "for the 5th line you need to print 5 stars right?\n",
+    "do you see where I'm getting at...? :)\n",
+    "\n",
+    "Good luck!\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "15589127-77c2-4518-870c-50e2d9fcb386",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "* \n",
+      "* * \n",
+      "* * * \n",
+      "* * * * \n",
+      "* * * * * \n",
+      "* * * * * * \n",
+      "* * * * * * * \n",
+      "* * * * * * * * \n",
+      "* * * * * * * * * \n",
+      "* * * * * * * * * * \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Solution:\n",
+    "n = 10\n",
+    "\n",
+    "for i in range(n):\n",
+    "    line_string = \"\"\n",
+    "    for j in range(i+1):\n",
+    "        line_string = line_string + \"*\" + \" \"\n",
+    "\n",
+    "    print(line_string)\n",
+    "        "
+   ]
```

Comment:
thanks for the PR! Could you please also add the answers for the bonuses? So, the participants can check later